yamls/hydra-yamls/SD-2-base-256.yaml

```diff
@@ -22,15 +20,14 @@ model:
   val_guidance_scales: []
   loss_bins: []
 dataset:
-  train_batch_size: ${batch_size}
-  eval_batch_size: 1024 # Should be 8 per device
+  train_batch_size: 2048 # Global training batch size
+  eval_batch_size: 1024  # Global evaluation batch size
   train_dataset:
     _target_: diffusion.datasets.laion.laion.build_streaming_laion_dataloader
```

Comment:
This might be a good chance to depreciate `build_streaming_laion_dataloader` and switch over to `build_streaming_image_caption_dataloader`. Could also be a separate PR though.